docs/installing.rst

```diff
@@ -62,6 +49,62 @@ In the previous two example dependencies are automatically resolved, if you simp
 
    $ git clone https://github.com/ThreatResponse/aws_ir.git
    $ cd aws_ir
+   $ virtualenv env -p python3
```

Comment:
Should we be consistent and align this virtualenv instruction with the above? python3 -m virtualenv env